examples/00_getting_started.py

```diff
@@ -0,0 +1,220 @@
+"""
+Getting Started
+===============
+
+The purpose of this guide is to provide an introduction to the functionalities of
+``skrub``, an open-source package that aims at bridging the gap between tabular
+data sources and machine-learning models. Please refer to our
+:ref:`installation_instructions` for installing ``skrub``.
+
+Much of ``skrub`` revolves around vectorizing, assembling, and encoding tabular data,
+to prepare data in a format that machine-learning models understand.
```

Comment:
The following is more of a "marketing/positioning" question for @GaelVaroquaux, should we add this?
```suggestion
to prepare data in a format that shallow or classic machine-learning models understand.
```

Reply:
Committing so that we can merge, we can always revert back or put more emphasis on this in other pages depending on the answer :)

Reply:
Yes, let's keep this discussion in mind for later :)